pom.xml

```diff
@@ -541,6 +543,9 @@
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-release-plugin</artifactId>
 					<version>${release.pluginVersion}</version>
+					<configuration>
+						<preparationGoals>clean exec:exec@update-maven-example-versions scm:add@checkin-updated-examples scm:checkin@checkin-updated-examples verify</preparationGoals>
```

Comment:
Additional `exec-maven-plugin` executions can be added if we acquire other kinds of examples.